src/query/config/src/config.rs

```diff
@@ -2774,6 +2774,14 @@ pub struct CacheConfig {
     )]
     pub inverted_index_info_count: u64,
 
+    /// Max bytes of cached inverted index filters used. Set it to 0 to disable it.
+    #[clap(
+        long = "cache-inverted-index-filter-size",
```

Comment:
Default is 17GB memory?

Reply:
It's just a temporary value, we need some tests it to determine how much memory is appropriate, we can change it to a smaller size first.